rickshaw-run

```diff
@@ -503,6 +503,10 @@ sub build_container_image {
                 }
             }
         }
+        if (! exists $run{'reg-tls-verify'}) {
+            $run{'reg-tls-verify'} = "true";
```

Comment:
enable by default?

Reply:
yes...it is actually the default behavior if you don't specify anything.